tests/e2e/specs/drawing.spec.js

```diff
@@ -0,0 +1,111 @@
+const mockResponse = { fileId: 'test', adminId: 'test' }
+
+describe('Drawing', () => {
+    function goToDrawing() {
+        cy.goToMapView()
+        cy.get('[data-cy="menu-button"]').click()
+        cy.get('.menu-section-head-title:first').click() // FIXME: how to address a specific menusection
+        cy.readStoreValue('state.ui.showDrawingOverlay').then((value) => {
+            expect(value).to.equal(true)
+        })
+    }
+
+    const tools = ['marker', 'text', 'line', 'measure']
+
+    function clickTool(name) {
+        expect(tools).to.include(name)
+        cy.get(`[data-cy="drawing-${name}`).click()
+        cy.readStoreValue('state.drawing.mode').then((value) => {
+            expect(value).to.equal(name.toUpperCase())
+        })
```

Comment:
Can be simplified as
```suggestion
        cy.readStoreValue('state.drawing.mode').should('eq', name.toUpperCase())
```